packages/dev/test-utils/src/events.ts

```diff
@@ -22,6 +22,22 @@ export function triggerPress(element, opts = {}) {
   fireEvent.click(element, {detail: 1, ...opts});
 }
 
+// Triggers a "touch" event on an element.
+export function triggerTouch(element, opts = {}) {
+  fireEvent.pointerDown(element, {pointerType: 'touch', ...opts});
+  fireEvent.pointerUp(element, {pointerType: 'touch', ...opts});
+}
+
+// Triggers a "longPress" event on an element.
+export const DEFAULT_LONG_PRESS_TIME = 500;
+
+export function triggerLongPress(element, opts = {}) {
+  fireEvent.pointerDown(element, {pointerType: 'touch', ...opts});
+  // @ts-ignore
+  act(() => jest.advanceTimersByTime(DEFAULT_LONG_PRESS_TIME));
```

Comment:
Don't need the ts-ignore, you can just change the line to be like so
```suggestion
  act(() => {
    jest.advanceTimersByTime(DEFAULT_LONG_PRESS_TIME);
  });
```

Reply:
Thanks, will fix later